client/src/components/Search.jsx

```diff
@@ -1,20 +1,20 @@
 import React from 'react';
-import styles from './Search.css'
+import styles from './Search.css';
 class Search extends React.Component {
-  constructor(props) {
-    super(props);
-  }
-
-
-
   render() {
     return (
-      <form className="search">
-        <input className="input-bar" onChange={this.props.searchBarTextChange} type="search" placeholder="Search reviews"></input>
-        <input className="input-button" onClick={this.props.searchSubmit} type="submit" value="Search Reviews"></input>
+      <form className="search" onSubmit={this.props.searchSubmit}>
+      <div>
+        <i class="fas fa-search"></i>
```

Comment:
Does the `class` attribute work here? I thought React only uses `className`.
I also see two classnames here but the acronym the second name forms would be the same as the first classname. I'm not sure if they were originally intended to be the same or if they are different things

Reply:
I'm using a library called font-awesome to render magnifying glass, and this is the line that allows for that. If I use className, it checks my Search.css instead of the package node modules, so this is the correct syntax. Thanks for spotting the potential error though!